frontend/lib/src/services/user_profile_service.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:io';
```

Comment:
I think we can transfer this insider `user_service.dart`

No need to create a new service.